src/javascript/app/pages/trade/tradepage.js

```diff
@@ -57,6 +58,21 @@ const TradePage = (() => {
                     if ($currency.next().attr('id') === $currency.attr('id')) $currency.next().eq(0).remove();
                 }
             });
+
+            BinarySocket.wait('website_status', 'landing_company').then(() => {
+                const is_logged_in = Client.isLoggedIn();
+                const is_eu = isEuCountry();
+                const is_malta = State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest';
+
+                if ((!is_logged_in && is_eu) || (is_logged_in && is_malta)) {
+                    if ((is_logged_in && is_eu) || (is_logged_in && is_malta)) {
```

Comment:
no, we only want to show this CTA to MF accounts, meaning that the boolean `is_maltainvest` is enough, as it will return false for logged out clients and only be returning true for authorized MF clients.